From f089fe5e75089237cf9951860f60ffd7a94857f8 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 09 May 2024 18:50:38 +0300
Subject: [PATCH] [SW-15774] board: fs1: support full path in its

Supports fs1 specific zephyr's its file to load files no located
under the same folder

Change-Id: I610a1bed13b3159aebf7b281bc319f115352104a
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/fs1_arc/fs1-zephyr.its b/board/habanalabs/fs1_arc/fs1-zephyr.its
index d694e41..ed8daf5 100644
--- a/board/habanalabs/fs1_arc/fs1-zephyr.its
+++ b/board/habanalabs/fs1_arc/fs1-zephyr.its
@@ -9,7 +9,7 @@
 	images {
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -23,7 +23,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -36,7 +36,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -50,7 +50,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
diff --git a/board/habanalabs/gaudi3/fake_fuse.its b/board/habanalabs/gaudi3/fake_fuse.its
index 2cd7a96..42fdc2a 100644
--- a/board/habanalabs/gaudi3/fake_fuse.its
+++ b/board/habanalabs/gaudi3/fake_fuse.its
@@ -11,7 +11,7 @@
 	images {
 		fake-fuse-image {
 			description = "Habana fake-fuse";
-			data = /incbin/("@DEV_PREFIX@-fake-fuse.bin.be");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-fake-fuse.bin.be");
 			compression = "none";
 			arch = "arc";
 			/* offset for fake fuse sector */
